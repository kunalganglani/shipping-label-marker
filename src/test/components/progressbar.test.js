import React from 'react';
import renderer from 'react-test-renderer';
import ProgressBar from '../../core/components/wizard/progress-bar/progress-bar';
test('ProgressBar renders correctly', () => {
  const props = {
    progress: 10,
  };
  const tree = renderer
    .create(<ProgressBar
      progress={props.progress}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
