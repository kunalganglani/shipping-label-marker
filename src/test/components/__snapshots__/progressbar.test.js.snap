// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar renders correctly 1`] = `
<div
  aria-valuemax={100}
  aria-valuemin={0}
  aria-valuenow={10}
  className="MuiLinearProgress-root MuiLinearProgress-colorPrimary MuiLinearProgress-determinate"
  role="progressbar"
>
  <div
    className="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar1Determinate"
    style={
      Object {
        "transform": "translateX(-90%)",
      }
    }
  />
</div>
`;
